service/db.go

```diff
@@ -569,3 +575,25 @@ func pairTokenWithSecret(token string) (*pblib.Identification, error) {
 
 	return nil, consts.ErrNoExistingTokenFound
 }
+
+// hasActiveSecret checks the database for is_active set to true in secrets table
+// Returns true if found, false otherwise, or any error encountered with the db itself
+func hasActiveSecret() (bool, error) {
+	command := `SELECT EXISTS( 
+  					SELECT 1 
+  					FROM user_security.secret 
+  					WHERE is_active = TRUE
+  				)`
+
```

Comment:
I think we should modify `deactivateSecret` to just deactivate everything that is `is_active = TRUE` instead of the following:
```

func deactivateSecret(secretKey string) error {
	if secretKey == "" {
		return nil
	}
	command := `UPDATE user_security.secret 
				SET	is_active = $1 
				WHERE secret_key = $2
				`
	_, err := postgresDB.Exec(command, false, secretKey)
	if err != nil {
		return err
	}
	return nil
```

Reply:
This is to prevent more than 1 active secret

Reply:
What do you think?

Reply:
thus rename this to `validateActiveSecret`

Reply:
discussed in slack